app/src/main/java/com/example/paulsuarez/a28_filterable_lists/models/Constructor.java

```diff
@@ -0,0 +1,29 @@
+package com.example.paulsuarez.a28_filterable_lists.models;
+
+import android.content.Intent;
+
+public class Constructor {
```

Comment:
This class should really have a more descriptive name. "Constructor" doesn't tell me anything about what the class is for.